backend/backend.go

```diff
@@ -446,25 +457,39 @@ func (backend *Backend) createAndAddBTCAccount(
 		)
 		signingConfigurations = append(signingConfigurations, signingConfiguration)
 	}
-	backend.createAndAddAccount(coin, code, name, signingConfigurations)
+	rootFingerprint, err := keystore.RootFingerprint()
+	if err != nil {
+		log.WithError(err).Error("Could not retrieve the keystore's root fingerprint")
+		return
+	}
+	err = backend.persistAccount(config.Account{
+		CoinCode:                coin.Code(),
+		Name:                    name,
+		Code:                    code,
+		SupportsUnifiedAccounts: keystore.SupportsUnifiedAccounts(),
+		RootFingerprint:         rootFingerprint,
+		Configurations:          signingConfigurations,
+	})
+
+	if errp.Cause(err) == ErrAccountAlreadyExists {
+		// This is adding initial default accounts. If this already happened, there is nothing else
```

Comment:
This might be temporary - it is not checking if the accounts exist by using the fingerprint yet, but fetches and uses the xpubs instead.